tools/gluten-te/ubuntu/dockerfile-buildenv

```diff
@@ -102,10 +102,14 @@ RUN set-login-env "LANG=en_US.UTF-8"
 RUN set-login-env "LANGUAGE=en_US:en"
 RUN set-login-env "LC_ALL=en_US.UTF-8"
 
-# Install Spark 3.2.2
+# Build & install Spark 3.2.2
```

Comment:
Do we want to pass-in Spark version as a docker build arg in  ```buildenv.sh```?

Reply:
As spark 3.2, 3.3 CI job shared a same docker image, we may just keep it here. Thanks!